init-service/src/test/java/uk/ac/ox/ndph/mts/init_service/service/RoleServiceTest.java

```diff
@@ -11,11 +11,11 @@
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.web.reactive.function.client.WebClient;
-import uk.ac.ox.ndph.mts.init_service.configuration.WebClientConfig;
 import uk.ac.ox.ndph.mts.init_service.config.AzureTokenService;
 import uk.ac.ox.ndph.mts.init_service.exception.DependentServiceException;
 import uk.ac.ox.ndph.mts.init_service.model.Permission;
 import uk.ac.ox.ndph.mts.init_service.model.Role;
+import uk.ac.ox.ndph.mts.roleserviceclient.configuration.WebClientConfig;
```

Comment:
Same as above

Reply:
In the next PR  uk.ac.ox.ndph.mts.init_service.configuration.WebClientConfig is going to be removed any way...

Reply:
ok